_posts/2018-04-25-noops.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: What the Ops is NoOps?
+author: jake_hall
+---
+
+Our industry, software engineering, is not shy about inventing new terms to describe what new things they're doing.
```

Comment:
Software engineering is perhaps a little too narrow? It implies that it's our only industry but I think we're broader than that.

Reply:
Removed!